src/window.rs

```diff
@@ -1396,6 +1396,19 @@ impl Window {
             .maybe_wait_on_main(|w| w.drag_resize_window(direction))
     }
 
+    /// Show window menu at current mouse position.
+    ///
+    /// There's no guarantee that calling this method show the window menu.
+    /// 
+    /// ## Platform-specific
+    /// - **Windows:** Window need to have `WS_SYSMENU` window style to be able to show the [`window menu`].
+    /// - **Android / iOS / macOS / Orbital / Wayland / Web / X11:** Unsupported.
```

Comment:
```suggestion
    /// **Android / iOS / macOS / Orbital / Wayland / Web / X11:** Unsupported.
```
If Winit doesn't have an API to remove `WS_SYSMENU`, I think this would become an implementation detail. If I understand this correctly, right now, showing the window menu will always work on Windows.